bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -597,6 +598,65 @@ func deleteDeployment(deploymentmanagerService *deploymentmanager.Service, ctx c
 	return nil
 }
 
+// Delete endpoint service from resources.
+func (gcp *Gcp) deleteEndpoints(ctx context.Context) error {
+	client, clientErr := google.DefaultClient(ctx, servicemanagement.ServiceManagementScope)
```

Comment:
Now you can get client from gcp directly.

Reply:
done.